grammars/asciidoc.cson

```diff
@@ -64,17 +80,123 @@
     'begin': '(^/{4,}$)'
     'end': '\\1'
     'name': 'comment.block.asciidoc'
+    'patterns': [
+      { 'include': '#inline' }
+    ]
+  }
+
+  # Matches example block
+  #
+  # Examples
+  #
+  #   ====
+  #   A multi-line example.
+  #
+  #   Notice it's a delimited block.
+  #   ====
+  {
+    'begin': '^(={4,})+?$'
+    'end': '\\1'
+    'name': 'markup.block.example.asciidoc'
+    'patterns': [
+      { 'include': '#inline' }
+    ]
+  }
+
+  # Matches sidebar block
+  #
+  # Examples
+  #
+  # ***********************************
+  # As an aside, I love Asciidoc
+  # ***********************************
+  {
+    'begin': '(^\\*{4,}$)'
+    'end': '\\1'
+    'name': 'markup.block.sidebar.asciidoc'
+    'patterns': [
+      { 'include': '#inline' }
+    ]
+  }
+
+  # Matches literal block
+  #
+  # Examples
+  #
+  # ...................................
+  # Daleks EXTERMINATE in monospace!
+  # ...................................
+  {
+    'begin': '(^\\.{4,}$)'
+    'end': '\\1'
+    'name': 'markup.block.literal.asciidoc'
+    'patterns': [
+      { 'include': [ '#inline', '#codeBlocks'] }
+    ]
+  }
+
+  # Matches passthrough block
+  #
+  # Examples
+  #
+  # +++++++++++++++++++++++++++++++++++
+  # <s>Could be struck through</s>
+  # +++++++++++++++++++++++++++++++++++
+  {
+    'begin': '(^\\+{4,}$)'
+    'end': '\\1'
+    'name': 'markup.block.passthrough.asciidoc'
+    'patterns': [
+      { 'include': '#inline' }
+    ]
+  }
+
+  # Matches quote declaration
+  #
+  # Examples
+  #
+  # [quote]
+  # ____________________________________________________________________
+  # Huh?
+  # ____________________________________________________________________
+  #
+  # [quote, Homer Simpson]
+  # ____________________________________________________________________
+  # D'oh!
+  # ____________________________________________________________________
+  #
+  # [quote, Erwin Schrödinger, Sorry]
+  # ____________________________________________________________________
+  # I don't like it, and I'm sorry I ever had anything to do with it.
+  # ____________________________________________________________________
+  {
+    'begin': '^\\[(quote|verse)(, ([\\w ]+))?(, ([\\w ]+))?'
```

Comment:
I see your point, we could remove this. The style I submitted only cares about markup.quote. I chose to submit it because without those lines, we're ignoring information that other style writers could potentially use. It's your call, just give the word and I'll remove it.